lib/rules/block-opening-brace-empty-line-before/README.md

```diff
@@ -0,0 +1,180 @@
+# block-opening-brace-empty-line-before
+
+Require or disallow an empty line before the opening brace of blocks.
+
+```css
+
+ * This line */
+/**  ↓
+a /* ↓ */
+/* ← */
+{
+  color: pink;
+}
+```
+
+## Options
+
+`string`: `"always" | "always-single-line" | "always-multi-line" | "never-single-line" | "never-multi-line" | "never"`
```

Comment:
Do we need that much options? We prefer to have less options as possible and don't add options “just in case”.

I think `"always-multi-line"` and `"never"` is enough (same as [block-closing-brace-empty-line-before](https://stylelint.io/user-guide/rules/block-closing-brace-empty-line-before/)). @stylelint/contributors what do you think?

Reply:
I agree. Let's go for parity with the `block-closing-brace-empty-line-before` rule and its `"always-multi-line"` and `"never"` options.